requirements.txt

```diff
@@ -1,6 +1,5 @@
-googlemaps==2.5.1
-nltk==3.4.5
-pandas==0.25.1
-mysql-connector-python==8.0.18
-numpy==1.16.5
-scikit-learn==0.21.3
+entrypoints==0.3
+flake8==3.7.9
+mccabe==0.6.1
+pycodestyle==2.5.0
+pyflakes==2.1.1

```

Comment:
Why have the existing libraries been removed from requirements.txt?

Reply:
Also, are all the new libraries required?

Reply:
Oh sorry, the libraries must've been removed when I froze the requirements. I'll fix it asap.

Reply:
I installed flake8 in the env via pip. These were all the pre-default libraries that were installed with it. Should I remove the rest of them apart from flake8?

Reply:
Oh, no need to put the contents of `pip freeze` into requirements.txt; rather, just add `flake8==3.7.9` to existing `requirements.txt` 😄